docs/openstack/create-and-connect-to-the-VM/wireguard/index.md

```diff
@@ -17,6 +17,12 @@ Ubuntu OS image, named "**wireguard_server**" on OpenStack, with
 
 Also, attach a Floating IP to this instance so you can ssh into it from outside.
 
+Create a new Security Group that is listening on **UDP port 51820** as shown below:
+
+![WireGuard Security Rule](images/wireguard_security_rule.png)
+
+All Security Groups attached to the WireGuard Server looks like as shown below:
```

Comment:
The security Groups attached to the WireGuard server should look similar to the image below:

(we might also want to list those security group names in text here as well)